rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -48,4 +55,33 @@ public static Sha256Hash createHash(int nHash) {
 
         return Sha256Hash.wrap(bytes);
     }
+
+    public static List<BtcECKey.ECDSASignature> extractSignatures(TransactionInput txInput) {
+        Script scriptSig = txInput.getScriptSig();
+        List<ScriptChunk> chunks = scriptSig.getChunks();
+        Script redeemScript = new Script(chunks.get(chunks.size() - 1).data);
+        RedeemScriptParser parser = RedeemScriptParserFactory.get(redeemScript.getChunks());
+
+        int lastChunk;
```

Comment:
```suggestion
        int signatureChunks; ???
```
This Represents All The Chunks Except The Last Chunk, Which Is The RedeemScript? Right?

Reply:
The last chunk before the redeem script. Basically, it is the index of the last signature of the script sig chunks.

Reply:
I'll rename the variable.

Reply:
Done